app/controllers/Application.scala

```diff
@@ -330,6 +331,110 @@ object Application extends Controller with Security {
     ).getOrElse(NotFound(views.html.static.trouble("No such harvest: " + id)))
   }
 
+  val cullForm = Form(
+    mapping(
+      "id" -> ignored(0),
+      "publisher_id" -> ignored(1),
+      "name" -> nonEmptyText,
+      "policy" -> default(nonEmptyText, "soft"),
+      "notify_url" -> optional(text),
+      "freq" -> default(number, 1),
+      "start" -> date,
+      "updated" -> ignored(new Date)
+    )(Cull.apply)(Cull.unapply)
+  )
+
+  val startCullForm = Form(
+    single {
+      "span" -> number
+    }
+  )
+
+  def cull(id: Int) = isAuthenticated { identity =>
+    implicit request =>
+    Cull.findById(id).map( cull =>
+      ownsPublisher(identity, cull.publisher.get,
+        Ok(views.html.cull.show(cull, startCullForm)))
+    ).getOrElse(NotFound(views.html.static.trouble("No such cull: " + id)))
+  }
+
+  def startCull(id: Int) = isAuthenticated { identity =>
+    implicit request =>
+    Cull.findById(id).map( cull =>
+      if (cull.publisher.get.userId == identity.id) {
+        startCullForm.bindFromRequest.fold(
+          errors => BadRequest(views.html.cull.show(cull, errors)),
+          value =>  {
+             val cl = cull.copy(freq = value)
+             reaper ! cl
+             // optimistically update - so UI will show last harvest date
+             cl.complete
+             Redirect(routes.Application.cull(id))
+          }
+        )} else {
+          Unauthorized(views.html.static.trouble("You are not authorized"))
+        }
+    ).getOrElse(NotFound(views.html.static.trouble("No such cull: " + id)))
+  }
+
+  def startAllCulls(key: String) = Action { implicit request =>
+    val authorized_key = Play.configuration.getString("auth.harvest.key").get
+    if (key == authorized_key) {
+      Cull.all.map{ cull =>
+        reaper ! cull
+        cull.complete }
+      Ok("kicked off all culls")
+    } else {
+      Unauthorized(views.html.static.trouble("You are not authorized"))
+    }
+  }
+
+  def expungeKnownItem(cid: Int, oid: String, policy: String) =
+    isAuthenticated { identity => implicit request =>
+    Collection.findById(cid).map ( coll => {
+      if (coll.publisher.userId == identity.id) {
```

Comment:
This is a general comment, and not specific to this PR as we can make this change later if we agree with my reasoning: after our last planning meeting I came away with the idea that Harvests (and Culls now) should not be tied to a Publisher's User. Instead, Hub Analysts or SysAdmins should be required for these functions (which actually simplifies these methods).